util/argo/argo.go

```diff
@@ -17,16 +17,14 @@ import (
 // ResolveServerNamespace resolves server and namespace to use given an application spec,
 // and a manifest response. It looks to explicit server/namespace overridden in the app CRD spec
 // and falls back to the server/namespace defined in the ksonnet environment
-func ResolveServerNamespace(destination *appv1.ApplicationDestination, manifestInfo *repository.ManifestResponse) (string, string) {
+func ResolveServerNamespace(destination appv1.ApplicationDestination, manifestInfo *repository.ManifestResponse) (string, string) {
```

Comment:
Do we need this function anymore? I think only Create app needs this logic.

Reply:
I searched usages of this and I think we should try removing this since we can now safely assume that server/namespace will be filled out in the object.

Reply:
Pushed an update that removes this function.